content/en/docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -117,17 +112,9 @@ etcd:
     <argument>: <value|string>
     <argument>: <value|string>
   image: <string>
-  serverCertSANs:
-  - <name1|string>
-  - <name2|string>
-  peerCertSANs:
-  - <name1|string>
-  - <name2|string>
 kubeProxy:
   config:
     mode: <value|string>
-    bindAddress: <address|string>
-    clusterCIDR: <cidr>
```

Comment:
[1] the PR looks good but i don't understand why it's removing the kubeProxy information that was added a couple of days ago?

Reply:
That may be an artifact of the rebasing. If so, please put it back.